src/timeline/TimelineModel.cpp

```diff
@@ -1067,9 +1067,9 @@ TimelineModel::openUserProfile(QString userid)
 }
 
 void
-TimelineModel::openRoomMembers()
+TimelineModel::openRoomMembers(QString room_id)
 {
-        MemberList *memberList = new MemberList(roomId());
+        MemberList *memberList = new MemberList(room_id == QString() ? roomId() : room_id, this);
```

Comment:
Can you change that to always require an explicit room id?

Reply:
Actually, I've been considering moving some of this room id-dependent stuff into TimelineViewManager. I'll probably do that instead

Reply:
The problem here is that I am considering having multiple rooms open at once. So everything should have the memberlist from the current room. In which case roomId() is probably right in this case and you shouldn't be able to pass a room id.

Reply:
(Or have a function in the timelinemanager, that can be passed a roomId, but not both)